<launch>
  <arg name="block_config" default="$(find block_printer_model_reader)/config/block.yaml"/>
  <arg name="block_model_filename" default=""/>
  <arg name="block_model_type" default=""/>
  <rosparam file="$(arg block_config)" command="load"/>
  <node pkg="block_printer_model_reader" name="block_printer_model_reader" type="block_printer_model_reader_node" output="screen">
    <param name="model_filename" type="string" value="$(arg block_model_filename)"/>
    <param name="model_type" type="string" value="$(arg block_model_type)"/>
  </node>
</launch>
